frontend/packages/shared/src/styles/abstracts/common/_typography.scss

```diff
@@ -22,14 +22,21 @@
   }
 }
 
-/* Font Weigh */
+/* Font Weight */
 .bold {
   &#{&} {
     font-weight: bold !important;
     padding-left: map-get($spacing, 'xxs');
   }
 }
 
+.lighter {
+  &#{&} {
+    color: $color-grey-600;
```

Comment:
Speksi taisi olla "harmaa 500", mutta nimen perusteella tämä ei ehkä ole sama? Voisi tarkistaa.

Reply:
Aa, okei. Mä taisin ymmärtää Oton pyynnön väärin. Luulin että tarkoitti että teksti harmaalla, mutta font weight 500. Mutta ilmeisesti meinaskin värinä juuri tätä: https://materialui.co/colors/grey/500

Reply:
Niin joo, voihan toi toki olla font-weight myös. Mut ehkä paras jos tarkistaa vielä mitä tarkoitettiin.

Reply:
Mä laitoin näitä kysymyksiä nyt lankaan joka tuolla auki (tiistailta klo 10.07)